saul-core/doc/SAULLANGUAGE.md

```diff
@@ -0,0 +1,77 @@
+## Classifiers
```

Comment:
This seems to be copied from `CONCEPTUALSTRUCTURE.md`. Did you mean to delete it from that file as well?